packages/workbox-webpack-plugin/src/lib/get-manifest-entries-from-compilation.ts

```diff
@@ -159,14 +181,14 @@ function filterAssets(compilation, config) {
     }
 
     // Next, check asset-level checks via includes/excludes:
-    const isExcluded = checkConditions(asset, compilation, config.exclude);
+    const isExcluded = checkConditions(asset, config.exclude);
```

Comment:
See previous comment—we might need to keep the old signature for backwards compatibiltiy.

Reply:
same as above